tests/test_butler.py

```diff
@@ -2478,6 +2507,63 @@ def test_fallback(self) -> None:
             butler.getURI(ref)
 
 
+@unittest.skipIf(create_test_server is None, "Server dependencies not installed.")
+class ButlerServerTests(ButlerTests, unittest.TestCase):
+    """Test RemoteButler and Butler server."""
+
+    configFile = None
+    predictionSupported = False
+
+    def setUp(self):
+        self.server_instance = self.enterContext(create_test_server(TESTDIR))
+
+    def tearDown(self):
+        pass
+
+    def create_butler(
+        self, run: str, storageClass: StorageClass | str, datasetTypeName: str
+    ) -> tuple[Butler, DatasetType]:
+        butler = self.server_instance.hybrid_butler._clone(run=run)
+        return self._setup_butler_data(butler, storageClass, datasetTypeName)
+
+    def create_empty_butler(self, run: str | None = None, writeable: bool | None = None):
+        return self.server_instance.hybrid_butler._clone(run=run)
+
+    def testConstructor(self):
+        # RemoteButler constructor is tested in test_server.py and
+        # test_remote_butler.py.
+        pass
+
+    def testDafButlerRepositories(self):
+        # Loading of RemoteButler via repository index is tested in
+        # test_server.py.
+        pass
+
+    def testGetDatasetTypes(self) -> None:
+        # This is mostly a test of validateConfiguration, which is for
+        # validating Datastore configuration and thus isn't relevant to
+        # RemoteButler.
+        pass
+
+    def testMakeRepo(self) -> None:
+        # Only applies to DirectButler.
+        pass
+
+    # Pickling not yet implemented for RemoteButler/HybridButler.
+    @unittest.expectedFailure
+    def testPickle(self) -> None:
+        return super().testPickle()
+
+    def testStringification(self) -> None:
+        # RemoteButler does not have datastore/registry strings like
+        # DirectButler does.
+        pass
```

Comment:
Can we have a bespoke test here that checks that RemoteButler stringifies into something reasonable? Maybe with the URL of the server?